Demo/src/main/java/com/paypal/android/ui/card/DateString.kt

```diff
@@ -0,0 +1,90 @@
+package com.paypal.android.ui.card
+
+import java.lang.Integer.min
+
+data class DateString(private val rawDate: String) {
```

Comment:
It'd be really cool to eventually move some of these helpers into a foundations module that can be shared across BT + PPCP + MXO!

Reply:
Def! Yeah that would be cool once we come up with some standard patterns on re-using Composables.

Reply:
+1!